jobclient/java/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -1279,6 +1309,9 @@ public static JSONObject jsonizeJob(Job job)
         if (job._datasets != null) {
             object.put("datasets", job._datasets);
         }
+        if (job._overrideGroupImmutability != null) {
+            object.put("override_group_immutability", job._overrideGroupImmutability);
```

Comment:
This String key name seems like it should be a separate static final variable?

Reply:
Yeah, but I'd rather keep the style consistent for the whole java client. We will probably want to refactor the whole thing later.